src/microbe_stage/Membrane.tscn

```diff
@@ -24,39 +25,101 @@ shader_parameter/albedoTexture = ExtResource("3")
 shader_parameter/damagedTexture = ExtResource("4")
 shader_parameter/dissolveTexture = ExtResource("5")
 
-[sub_resource type="ShaderMaterial" id="ShaderMaterial_p8cmc"]
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_2"]
 resource_local_to_scene = true
 render_priority = 0
-shader = ExtResource("6_vljlr")
+shader = ExtResource("6")
 shader_parameter/wigglyNess = 1.0
 shader_parameter/movementWigglyNess = 1.0
 shader_parameter/waviness = 40.0
 shader_parameter/waveSpeed = 10.0
 shader_parameter/fade = 0.2
 shader_parameter/tint = Color(0, 0.55, 0.8, 1)
 
-[sub_resource type="ShaderMaterial" id="ShaderMaterial_lboqi"]
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_3"]
 resource_local_to_scene = true
 render_priority = 0
-shader = ExtResource("7_grw0d")
+shader = ExtResource("7")
 shader_parameter/wigglyNess = 1.0
 shader_parameter/movementWigglyNess = 1.0
 shader_parameter/fade = 0.25
 shader_parameter/tint = Color(0.4, 0.8, 0.6, 1)
 
+[sub_resource type="Gradient" id="Gradient_1"]
+offsets = PackedFloat32Array(0, 0.3, 0.7, 1)
+colors = PackedColorArray(1, 1, 1, 1, 0.8, 0.9, 1, 1, 0.9, 0.95, 1, 1, 1, 1, 1, 1)
+
+[sub_resource type="GradientTexture2D" id="GradientTexture2D_1"]
+gradient = SubResource("Gradient_1")
+fill = 1
+fill_from = Vector2(0.5, 0.5)
+fill_to = Vector2(1, 0.5)
+
+[sub_resource type="FastNoiseLite" id="FastNoiseLite_1"]
+frequency = 0.005
+fractal_octaves = 2
+
+[sub_resource type="NoiseTexture2D" id="NoiseTexture2D_1"]
+seamless = true
+seamless_blend_skirt = 0.4
+noise = SubResource("FastNoiseLite_1")
+
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_bubble"]
+render_priority = 0
+shader = ExtResource("8")
+shader_parameter/smoothness = 0.5
+shader_parameter/refraction_strength = 0.2
+shader_parameter/bubble_noise = SubResource("NoiseTexture2D_1")
+shader_parameter/bubble_gradient = SubResource("GradientTexture2D_1")
+
+[sub_resource type="SphereMesh" id="SphereMesh_1"]
+material = SubResource("ShaderMaterial_bubble")
+radius = 0.9
+height = 1.8
+radial_segments = 32
+rings = 24
+
+[sub_resource type="Curve" id="Curve_1"]
+_data = [Vector2(0, 0.3), 0.0, 2.0, 0, 0, Vector2(1, 1), 2.0, 0.0, 0, 0]
+point_count = 2
+
 [node name="Membrane" type="MeshInstance3D" node_paths=PackedStringArray("engulfAnimationMeshInstance", "mucocystAnimationMeshInstance")]
-process_priority = 2
-cast_shadow = 0
```

Comment:
You have changed script order priority and shadow settings for the membrane. If these are intentional, could you explain why these are necessary?